extern/grplib-4.7/configure

```diff
@@ -13898,7 +13898,8 @@ $debug ||
 if test -n "$CONFIG_FILES"; then
 
 
-ac_cr='
'
+ac_cr='
+'
```

Comment:
I just noticed git automatically removed a spurious CR (Carriage Return ^M) character from here. This should not have any effect on the scripts, but I'll double check.